src/services/posts/record-likes.js

```diff
@@ -0,0 +1,27 @@
+const db = require("../../db")
+const {
+	GifHasAlreadyBeenLikedError
+} = require("../errors")
+const customError = require("../../lib/custom-error")
+
+const uniqueErrorCode = '23505'
+
+const likePost = async({userId, id}) => {
```

Comment:
let's make `id` `postId` to make it clear what it means

Reply:
yes please